include/js/general.js

```diff
@@ -7718,6 +7718,21 @@ function cbdzdragFinish() {
 	document.getElementById('corebosdropzonemsg').style.display = 'none';
 }
 
+async function downloadFilesAsZip(fileName, urls) {
+	var baseurl = `index.php?module=Utilities&action=UtilitiesAjax&file=ExecuteFunctions&functiontocall=compressAndDownloadFiles&zipFileName=test.zip&filesToBeCompressed=${JSON.stringify(urls)}`;
```

Comment:
just to be sure you know that you can call this with the [ExecuteFunctions](https://github.com/tsolucio/corebos/blob/master/include/js/vtlib.js#L404) function.